sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -7,36 +7,96 @@
 namespace Azure.AI.FormRecognizer.Training
 {
     /// <summary>
+    /// Represents a custom model returned from training.
     /// </summary>
     public class CustomFormModel
     {
-        internal CustomFormModel()
+        internal CustomFormModel(Model_internal model)
         {
+            ModelId = model.ModelInfo.ModelId.ToString();
+            Status = model.ModelInfo.Status;
+            CreatedOn = model.ModelInfo.CreatedDateTime;
+            LastModified = model.ModelInfo.LastUpdatedDateTime;
+            Models = ConvertToSubmodel(model);
+            TrainingDocuments = model.TrainResult.TrainingDocuments;
```

Comment:
It looks like Errors may be missing?

Reply:
Yes. I saw https://github.com/Azure/azure-sdk-for-net/issues/11181 and wanted to understand more before doing it.
I will address it in a different PR

Reply:
The TrainResult is not a required field and can be null. Just want to make sure that's accounted for in this code

Reply:
Goot catch! thank you.
Fixed